src/codec_svt.c

```diff
@@ -85,8 +87,12 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         svt_config->logical_processors = encoder->maxThreads;
         svt_config->enable_adaptive_quantization = AVIF_FALSE;
         // disable 2-pass
+#if SVT_VERSION_PATCHLEVEL >= 8
```

Comment:
I think this test should read

`#if SVT_CHECK_VERSION(0, 8, 8)`

Reply:
No problem but in my opinion it isn't future-proof. It will work badly for 0.8.9.
This is just unapproved information. I also don't know if it makes sense to put `#if` since cmd version is defined.
I will add cmd changes after approving version 0.8.8.